src/main/java/core/basesyntax/FigureDescription.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface FigureDescription {
+    String getRandomParameters();
+
```

Comment:
```suggestion
    String draw();
```

why getRandomParameters?